File: src/InstantMessenger.Web/ClientApp/src/app/home/groups/services/responses/group.dto.ts
export type GroupId = string;
export interface GroupDto {
  groupId: GroupId;
  name: string;
  createdAt: string;
  ownerId: string;
}

export interface ChannelDto {
  channelId: string;
  groupId: string;
  channelName: string;
}

File: src/Modules/InstantMessenger.Groups/Application/ResponseDtos/GroupDto.cs
using System;

namespace InstantMessenger.Groups.Application.ResponseDtos
{
    public class GroupDto
    {
        public Guid GroupId { get; set; }
        public string Name { get; set; }
        public DateTimeOffset CreatedAt { get; set; }
        public Guid OwnerId { get; set; }

        public GroupDto(Guid groupId, string name, DateTimeOffset createdAt, Guid ownerId)
        {
            GroupId = groupId;
            Name = name;
            CreatedAt = createdAt;
            OwnerId = ownerId;
        }
        public GroupDto(){}
    }
}